swagger.yaml

```diff
@@ -155,5 +317,116 @@ components:
           format: date
         assignedPersons:
           type: array
-          items: 
-            type: integer
\ No newline at end of file
+          items:
+            type: integer
+
+    Leads:
+      type: object
+      description: Lead data from Pipedrive API
```

Comment:
Before listing the properties you should define which ones are required, I suspect they are all required.You can see an example of how this is done in this spec: https://github.com/Metatavu/time-bank-api-spec/blob/e8b5222db8ad2762c5fc39cd11206ccc85a97026/swagger.yaml